docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,33 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us bridge smart contracts to a frontend. Internally, we call this position "Glue". Our Web3 Developer is ideally not only a Javascript expert but also a decentralization evangelist—someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript interface between smart contracts and the native web experience.
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/), [ethers.js](https://github.com/ethers-io/ethers.js/), [Parity's 007 Bonds and [ethjs](https://github.com/paritytech/oo7-parity) and [eth.js](https://github.com/ethjs/ethjs).
```

Comment:
One thing: I think this is broken; it should just be:

`[Parity's 007 Bonds](https://github.com/paritytech/oo7-parity), and [eth.js](https://github.com/ethjs/ethjs).`